### PR TITLE
(more) documentation changes

### PR DESCRIPTION
**Pull request by alcohol**
refs #4171

### CODE REVIEW

**Comment by Seldaek** [comment]
Thanks, great overhaul :) Just had to fix a couple things for rendering, I hope I didn't miss anything.

File: doc/00-intro.md [hunk 1]
```diff
@@ -1,54 +1,41 @@
...
 ## Dependency management
 
-Composer is not a package manager. Yes, it deals with "packages" or libraries, but
-it manages them on a per-project basis, installing them in a directory (e.g. `vendor`)
-inside your project. By default it will never install anything globally. Thus,
-it is a dependency manager.
+Composer is **not** a package manager in the same sense as Yum or Apt are. Yes,
+it deals with "packages" or libraries, but it manages them on a per-project
+basis, installing them in a directory (e.g. `vendor`) inside your project. By
+default it will never install anything globally. Thus, it is a dependency
```
Comment (staabm) [inline doc/00-intro.md]: Hmm never installs globally is actually not true, https://getcomposer.org/doc/03-cli.md#global

File: doc/00-intro.md [hunk 2]
```diff
@@ -1,54 +1,41 @@
...
 b) Some of those libraries depend on other libraries.
 
-c) You declare the things you depend on.
+Composer:
 
-d) Composer finds out which versions of which packages need to be installed, and
-   installs them (meaning it downloads them into your project).
-
-## Declaring dependencies
+c) Enables you to declare the libraries you depend on.
```
Comment (staabm) [inline doc/00-intro.md]: Resume the list of the `Suppose` part doesnt make sense?

File: doc/00-intro.md (hunk 1 above)
Comment (alcohol) [inline doc/00-intro.md]: > by default

`global` is not the default command.

Comment (staabm) [inline doc/00-intro.md]: Oh, missed that part. You are right.

**Pull request merged**